electron/generators/diograph-generator.js

```diff
@@ -1,40 +1,43 @@
 const { isEmpty, promiseAllReduce } = require('../lib/utils')
-const { readPaths } = require('../readers/folder-reader')
+const { readFolderContents } = require('../readers/folder-reader')
 const { generateFileDiory, generateFolderDiory } = require('./diory-generator')
 const { generateFileLink, generateFolderLink } = require('./link-generator')
 
-function getDiorys(diograph, links) {
-  return Object.values(links).map(({ id }) => diograph[id])
+function getDioryDiographs(diograph, dioryIds) {
+  return dioryIds.map((dioryId) => diograph[dioryId])
 }
 
-function addToDiograph(diory) {
-  return (
-    diory.id && {
-      [diory.id]: diory,
-    }
-  )
-}
+// MOVED TO LINK GENERATOR
```

Comment:
this is not link related; just diograph related

Reply:
link generator is like: 
{
  [path] : { id }
}

Reply:
Miten se sitten voi olla niin samannäköinen kuin noi generate-link funkkarit? :)

Reply:
siis niin että kerätään array objektin sijaan; varmaan mahdollista, mutta en ole varma mitä hyötyä ja haittoj tulisi; pitäikö silloin huolehtia, että ei tule duplikaatteja yms. voihan tuota kokeilla, jos selventäisi!